dist/style.scss

```diff
@@ -468,6 +468,126 @@ line-height: 24px;
   }
 }
 
+#contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  .contact-text {
+    display: flex;
+    // align-items: center;
+    // border: 3px solid red;
+    width: 327px;
+    height: 216px;
+    margin: 5rem 1rem 3rem 0;
+    p {
+      font-family: $inter;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 32px;
+      line-height: 44px;
+      color: $base-navy;
+    }
+  }
+}
+
+.form {
+  margin-top: 10rem;
+  fieldset {
+    width: 384px;
+    height: 350px;
+    position: relative;
+    border: none;
+    .input-group {
+      width: 100%;
+      input {
+        height: 35px;
+      }
+
+      input,
+      textarea {
+        margin-left: 1rem;
+        width: 100%;
+        border: none;
+        border-bottom: 1px solid $gray3;
+      }
+
+      input:focus,
+      textarea:focus {
+        outline: none;
+      }
+
+      textarea {
+        background-color: #e7e5e5;
+      }
+
+      .submitErr {
+        color: red;
+      }
+      .fa-sharp {
+        color: green;
+      }
+
+      .firstNerror {
+        position: absolute;
+        right: 360px;
+        top: 10px;
+      }
+
+      .lastNerror {
+        position: absolute;
+        right: 15px;
+        top: 10px;
+      }
+
+      .fullNerror {
+        position: absolute;
+        right: 20px;
+        top: 10px;
+      }
+
+      .emailError {
+        position: absolute;
+        right: 20px;
+        top: 45px;
+      }
+
+      .msgError {
+        position: absolute;
+        right: 20px;
+        top: 200px;
+      }
+
+      .fullN,
+      #fullName {
+        display: block;
+      }
+
+      .err {
+        color: red;
+      }
+    }
+    .allNames {
+      display: none;
+    }
+    button {
+      margin-top: 1.5rem;
+      width: 122px;
+      height: 48px;
+      background-color: $primary;
+      color: $base-white;
+      border: none;
+      font-family: $roboto;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 17px;
+      line-height: 24px;
+      margin-left: 1rem;
```

Comment:
- [ ] Nice job of designing a good button +1. however, It would be better to align the button in the **middle** to match that of Figma design,  
- [ ] it would be better too to add some button interactions such as ```hover```, ```active``` (when it's pressed), ```disabled``` and ```enable``` (default state) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27).